cmd/cli/completion.go

```diff
@@ -35,7 +35,11 @@ func newOAIClients() (oaiClients, error) {
 	var err error
 
 	if azureOpenAIEndpoint == nil || *azureOpenAIEndpoint == "" {
-		oaiClient = openai.NewClient(*openAIAPIKey)
+		if *openAIBase == "" {
```

Comment:
instead of a conditional here, can we set the default `openAIBase` to be `https://api.openai.com/v1` and user can override if they wanted to?

Reply:
Thank you for the review. It has been modified.